WordPress/src/main/res/layout/wpwebview_activity.xml

```diff
@@ -45,10 +45,10 @@
                 android:layout_height="match_parent" />
 
             <include
+                android:id="@+id/progress_layout"
                 layout="@layout/progress_layout"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:layout_above="@+id/navbar_container"
```

Comment:
AS marked this as wrong because the referenced view is in another Relative layout